viz.yaml

```diff
@@ -18,7 +18,7 @@ info:
       version: 0.9-7
     dataRetrieval:
       repo: CRAN
-      version: 2.7.4
+      version: 2.7.3
```

Comment:
does the older version work better?

Reply:
It was just to avoid having to set package installs to GRAN, since CRAN only has 2.7.3

Reply:
oh, OK